include/real/real.hpp

```diff
@@ -103,11 +106,16 @@ namespace boost {
                 // Internal number to iterate
                 real const* _real_ptr = nullptr;
 
+                std::variant<boost::real::real_explicit::const_precision_iterator,
+                    boost::real::real_algorithm::const_precision_iterator> diff_it;
+
+/*                
```

Comment:
Don't leave dead commented code around. If you removing some code, just delete it. We have git to find out how it was before if we ever want to revive it.